salesforce_api/gsheet-new-row-to-sfdc-new-contact/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+template = true
+org = "choreo"
```

Comment:
Org should be changed to `integration_samples`. That is what we have used earlier.

Reply:
Updated in [86c29e1](https://github.com/ballerina-guides/integration-samples/pull/31/commits/86c29e1236dcc64dd4c790e62e2633241a8a8307)